packages/@aws-cdk/cloud-assembly-schema/README.md

```diff
@@ -0,0 +1,12 @@
+## Cloud Executable protocol
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+
+---
+<!--END STABILITY BANNER-->
+
+This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
```

Comment:
This is where you should be describing the protocol, i.e., (1) what is it, (2) why does it exist, (3) what is the  protocol and (4) (if applicable) how to modify/upgrade.

Can we do the same for the `cx-api` module? I suppose this one contains the exact schema (and low level APIs), while the other contains mechanisms to read and write this schema as a file. Did I get that right?

Reply:
Yeap yeap, i'll do that 👍